[HID] Enable Keyboard Passthru

Gliniak opened the pull request.

chrisps commented on src/xenia/hid/winkey/winkey_input_driver.cc:
```diff
@@ -25,10 +25,29 @@
 namespace hid {
 namespace winkey {
 
+bool __inline IsPassThruForUserEnabled(uint32_t user_index) {
```
can't this just be user_index == cvars::keyboard_passthru_user_index
is -1 possible? it doesnt seem like it should be

Gliniak replied:
Yeah it shouldn't. -1 is used but as 0xFF to define any user

nickcool2901 commented:
keyboard_passthru isnt in any new builds

Rackover commented:
Would love to see merged in, is there any problem with the PR that is currently delaying it ?

Gliniak commented:
> Would love to see merged in, is there any problem with the PR that is currently delaying it ?

Yes, It's not fully functional and doesn't work in some cases

Rackover commented:
I guess nothing ever is, but it's a start! If it's not breaking any other functionality in the emulator, maybe this could get merged in and improved later? 
Of course if it's breaking existing functionality it should not, but maybe it's worth having if it's not.

Gliniak committed:
[HID] Enable Keyboard Passthru

Thanks Adrian for figuring out missing XINPUT_KEYSTROKE_VALIDUNICODE flag.
Thanks Clippy95 for figuring out shift and capslock

ItsJokerZz commented:
How do you enable? Downloaded the PR build `f6ae651_canary_experimental` and see no option...
